tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
```

Comment:
I was hoping that we could include some stricter type checking into our tsconfig.
Could you check wheater or not setting  strict mode to true will cause us any issues at this stage of developement ? If not at least ```"noImplicitAny": true &&  "strictNullChecks": true``` would be nice to haves imho   
 
```
/* Strict Type-Checking Options */        
    "strict": true,                        /* Enable all strict type-checking options. */
    "noImplicitAny": true,                 /* Raise error on expressions and declarations with an implied 'any' type. */
    "strictNullChecks": true,              /* Enable strict null checks. */
    "noImplicitThis": true,                /* Raise error on 'this' expressions with an implied 'any' type. */
    "alwaysStrict": true,                  /* Parse in strict mode and emit "use strict" for each source file. */
```

Reply:
As the description implies, looks like `strict` enables all to the ones below it, I will add it.

Reply:
Pushed `strict` mode for now, will have a read tomorrow as to problems these options could cause.

Reply:
👍

Reply:
Had a bit of a search and most articles I can see about strict mode seem to encourage it. Let's leave it completely enabled for now and see how we get on.